### PR TITLE
Add Rank Functionality

### PR DESCRIPTION
**Pull request by rjawesome**
I haven't tested it so please test or at least read the code thoroughly before approving this or deploying to production.

### CODE REVIEW

File: main.go [hunk 1]
```diff
@@ -7,6 +7,7 @@ import (
 	"net/http"
 	"strconv"
 	"time"
+	"html/template"
```
Comment (sourque) [inline main.go]: Hmm hasn't been gofumpt'd

File: main.go [hunk 2]
```diff
@@ -23,13 +24,23 @@ func init() {
...
 	readConfig(&sarpConfig)
 	checkConfig()
 
 	// Initialize Gin router
 	// gin.SetMode(gin.ReleaseMode)
 	r := gin.Default()
+
+	//Add Increment Function to Router
+	r.SetFuncMap(template.FuncMap{
+		"increment": increment,
```
Comment (sourque) [inline main.go]: We could avoid having the function explicitly defined for everyone, and instead do something like:
```
	// Add... add function
	r.SetFuncMap(template.FuncMap{
		"increment": func(x int) int {
			return x + 1
		},
	})
```

**Review by sourque — COMMENTED** [review]
Looks really good overall!

File: main.go (hunk 2 above)
Comment (rjawesome) [reply main.go]: Makes sense. Got it!

**Review by safinsingh — APPROVED** [review]
Looks good to me! I think @sourque should be the one to merge this.

**Pull request merged**